src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -50,8 +50,10 @@ if [ "$(uname -s)" == "Darwin" ]; then
   # On OS X, we will enable core dump generation only if there are no core 
   # files already in /cores/ at this point. This is being done to prevent
   # inadvertently flooding the CI machines with dumps.
-  if [ ! "$(ls -A /cores)" ]; then 
-    ulimit -c unlimited
+  if [ -d "/cores" ]; then
```

Comment:
The original code was checking there are no dump files already. If /core does not exist then there are no dump files already. So it should run ulimit.
As you have wrutten it it will NOT run ulimit if /cores does not exist?